src/main/java/com/exasol/releaserobot/github/GithubRepositoryGateway.java

```diff
@@ -22,10 +22,11 @@ public GithubRepositoryGateway(final GithubGateway githubGateway) {
     @Override
     public Repository getRepositoryWithBranch(final UserInput userInput) throws GitHubException {
         if (userInput.hasBranch()) {
-            return this.githubGateway.getRepositoryWithUserSpecifiedBranch(userInput.getRepositoryFullName(),
+            return this.githubGateway.getRepositoryWithUserSpecifiedBranch(userInput.getRepositoryName(),
                     userInput.getBranch());
+        } else {
```

Comment:
you can remove this "else" line, makes the code more readable.

Reply:
In opposite, I think with else it's more readable :D